src/components/producerItems/components/ProducerItems.js

```diff
@@ -3,13 +3,17 @@ import React from 'react';
 import shallowCompare from 'shallow-equal/objects';
 import { bindActionCreators } from "redux";
 import { connect } from 'react-redux';
-import { Button, Icon } from 'antd';
+import { Button, Icon,  message } from 'antd';
 
 import ProductList from './ProductList';
 import ProductEditModal from './ProductEditModal';
 
-import { requestProductList, requestProductUpdate } from '../actions';
-import { getProductList } from '../selectors';
+import {
+    requestProductDelete,
+    requestProductList,
+    requestProductUpdate
+} from "../actions";
```

Comment:
import all the actions
```
import * as actions from 'actions';
```

Reply:
ok

Reply:
I am not using all the actions in this page, some(error and success) are been called from the saga.

Reply:
I understand but there are conflicting names in this file. Example: `requestProductDelete` on line 12 and `requestProductDelete` on line 94. these two functions are not the same thing. The one on line 94 is bound in `dispatch` and thus the redux store. Import all the actions and distructure `actions` in your `mapDispatchToProps` function